### ISSUE TITLE
Flutter Dev Tools as Chrome Extension

### ISSUE THREAD

**Issue by roman-petrov** [flutter/devtools#2930]
## Use case

React has [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) Chrome extension. In my opinion it is very comfortable to have developer tools right inside in Chrome Developer tools: this allows to see what is happening on the page without switching context.

## Proposal

As I know, Flutter Dev Tools is actually a Flutter Web application. Chrome extensions basically are web applications too. What do you think about packaging Flutter Dev Tools as Chrome extension? I suppose it might be helpful for Flutter Web development (and maybe for cross platform development too).

**Comment by grouma** [comment]
We are blocked on work from the Chrome team. Currently the debugger tab will steal focus from a custom extension tab while stepping and setting breakpoints.

**Comment by elliette** [comment]
Closing in favor of https://github.com/dart-lang/webdev/issues/2482 which has details on what is needed to support this for the Dart Debug Extension.
